tests/integ/conftest.py

```diff
@@ -11,9 +11,27 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
+
+import os
+
 import pytest
 
 
+def create_sagemaker_local_network_if_necessary():
+    """
+    Docker has a known race condition which allows two parallel processes
+    to create a duplicated networks with the same name. This function
+    creates the network sagemaker-local beforehand, avoiding this issue
+    in CI.
+    Returns:
+
+    """
+    os.system('docker network create sagemaker-local')
```

Comment:
how is this 'if necessary' ?
shouldnt this method be renamed to create_sagemaker_local_network() ?

Reply:
Good point. Done.